spine-cpp/spine-cpp/include/spine/SpineString.h

```diff
@@ -200,14 +202,15 @@ namespace spine {
 		}
 
 		~String() {
-			if (_buffer) {
+			if (_buffer && _tofree) {
 				SpineExtension::free(_buffer, __FILE__, __LINE__);
 			}
 		}
 
 	private:
 		mutable size_t _length;
 		mutable char *_buffer;
+		mutable bool _tofree;
```

Comment:
Needs a better name.

Reply:
Ya I was not actually not 100% sure on this...  I don't love the workaround, but we definitely need a way to interact with the spine api (which takes `spinestring`) without doing an additional alloc and memcopy for a stack temporary.   

Perhaps a better design would be to leave this class implementation alone and write a `SpineStringView` class which inherits `SpineString` (so we can pass them into the spine api), but the SpineStringView would have nothing to do with the backing character memory it's initialized with...  I don't know.